packages/guideline-blocks-settings/src/components/RichTextEditor/plugins/LinkPlugin/FloatingLink/CustomFloatingLink.tsx

```diff
@@ -55,17 +58,32 @@ export const CustomFloatingLink = () => {
 
     return (
         <>
-            {insertState.isOpen && !editState.isOpen && (
-                <div ref={insertRef} {...insertProps} style={{ ...insertProps.style, zIndex: 1000 }}>
-                    {input}
-                </div>
-            )}
+            {insertState.isOpen &&
+                !editState.isOpen &&
+                createPortal(
+                    <div
+                        data-is-underlay
+                        ref={insertRef}
+                        {...insertProps}
+                        style={{ ...insertProps.style, zIndex: 1000, ...BlockStyles[TextStyles.p] }}
```

Comment:
Since it is a `Portal` you don't need `zIndex`

Reply:
It is needed as otherwise elements with higher zIndex are layered on top of it.
<img width="859" alt="image" src="https://github.com/user-attachments/assets/b5d6b5f7-50be-4f57-a761-4ec5657174ea">

Reply:
Update: with the new isolation it is indeed no longer needed. Adjusted